volumes/csi-wrapper/README.md

```diff
@@ -0,0 +1,295 @@
+3. Create nginx peer-pod demo with with `podvm-wrapper` and `ibm-vpc-block-csi-driver` container
+```bash
+kubectl create -f hack/ibm/nginx-kata-with-my-pvc-and-csi-wrapper.yaml
+```
+
+4. Wait 2 minutes, check if the `nginx` pod is running:
+```bash
+kubectl get po -A | grep nginx
+kube-system   nginx                                                 3/3     Running             0              101s
+```
+
+5. Check the `PeerpodVolume` crd object and the state is `nodePublishVolumeApplied`
+```bash
+kubectl get PeerpodVolume -A
+NAMESPACE     NAME                                        AGE
+kube-system   r022-31f30b9b-556b-4a5d-80b0-419270a8783e   11m
+kubectl -n kube-system get PeerpodVolume r022-31f30b9b-556b-4a5d-80b0-419270a8783e -o yaml |grep state
+  state: nodePublishVolumeApplied
+```
+
+6. Exec into nginx container check the mount:
+```bash
+# kubectl exec nginx -n kube-system -c nginx -i -t -- sh
+# lsblk
+NAME    MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
+loop0     7:0    0 91.8M  1 loop
+loop1     7:1    0 63.2M  1 loop
+loop2     7:2    0 49.6M  1 loop
+vda     252:0    0  100G  0 disk
+|-vda1  252:1    0 99.9G  0 part /run/secrets/kubernetes.io/serviceaccount
+|-vda14 252:14   0    4M  0 part
+`-vda15 252:15   0  106M  0 part
+vdb     252:16   0  372K  0 disk
+vdc     252:32   0   44K  0 disk
+vdd     252:48   0   10G  0 disk /mount-path
+# ls /mount-path
+lost+found
+```
+**Note** We can see, the device **vdd** is mounted to `/mount-path` in nginx container as expected.
+
+- Create a file under `/mount-path`
+```bash
+echo "from nignx container date:" $(date) > /mount-path/incontainer.log
+```
+- On the development machine check the created vsi pod
+```bash
+ibmcloud is ins |grep nginx
+02e7_d9d5d4f4-d188-4c96-8f7b-c390b97f7aa4   podvm-nginx-1c0b9c12    running   10.244.0.4     -                 bx2-2x8    podvm-82ae9f8-dirty-amd64            liudali-csi-wrapper-vpc   jp-tok-1   default
+```
+- Check the netwokr interface id
+```bash
+ibmcloud is in 02e7_d9d5d4f4-d188-4c96-8f7b-c390b97f7aa4
+...
+Getting instance 02e7_d9d5d4f4-d188-4c96-8f7b-c390b97f7aa4 under account DaLi Liu's Account as user liudali@cn.ibm.com...
+
+ID                                    02e7_d9d5d4f4-d188-4c96-8f7b-c390b97f7aa4
+Name                                  podvm-nginx-1c0b9c12
+...
+Created                               2022-12-28T08:39:30-08:00
+Network Interfaces                    Interface   Name                              ID                                          Subnet                    Subnet ID                                   Floating IP   Security Groups                   Allow source IP spoofing   Reserved IP
+                                      Primary     snowless-defamed-dolphin-caller   02e7-663133fd-94aa-4d6b-bf2f-2163280dabf1   liudali-csi-wrapper-sn1   02e7-bf2a1bf2-5f94-4290-b422-b51f449a37a3   -             castle-duchess-regalia-dynamite   false                      10.244.0.4
+...
+```
+- Create one floating ip for the created vsi
```

Comment:
nit: `- With the network interface ID, create one floating ip for the pod VSI`

Reply:
fixed.